open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/open-lineage-connectors/open-lineage-janus-connector/src/main/java/org/odpi/openmetadata/openconnectors/governancedaemonconnectors/openlineageconnectors/janusconnector/graph/LineageGraphQueryHelper.java

```diff
@@ -101,12 +101,22 @@ public Set<LineageEdge> getLineageEdges(Graph subGraph) {
         Set<LineageEdge> lineageEdges = new HashSet<>();
         while (originalEdges.hasNext()) {
             Edge next = originalEdges.next();
-            LineageEdge newLineageEdge = new LineageEdge(next.label(), getNodeID(next.outVertex()), getNodeID(next.inVertex()));
+            LineageEdge newLineageEdge = new LineageEdge(getEdgeID(next), next.label(), getNodeID(next.outVertex()), getNodeID(next.inVertex()));
             lineageEdges.add(newLineageEdge);
         }
         return lineageEdges;
     }
 
+    private String getEdgeID(Edge edge) {
```

Comment:
I suggest some comments around this to explain the thinking behind this.

Reply:
Until now, edges were not sent to the client with their id. This method computes an edge id in a way similar to computing a node id